src/redux/actions/genes/LoadMarkerGenes.js

```diff
@@ -0,0 +1,47 @@
+import {
+  MARKER_GENES_ERROR, MARKER_GENES_LOADING,
+} from '../../actionTypes/genes';
+import sendWork from '../../../utils/sendWork';
+
+const REQUEST_TIMEOUT = 60;
+
+const loadMarkerGenes = (experimentId, resolution = null) => async (dispatch, getState) => {
```

Comment:
Remove `= null`

Reply:
Idea: on redux action creators assert the parameters that shouldn't be null aren't null.